packages/zowe-explorer/src/commands/UnixCommandHandler.ts

```diff
@@ -43,219 +38,181 @@ export class UnixCommandHandler extends ZoweCommandProvider {
+        issueCmdNotSupportedMsg: (profileType: string) =>
+            vscode.l10n.t({
+                message: "Issuing commands is not supported for this profile type, {0}.",
+                args: [profileType],
+                comment: ["Profile type"],
+            }),
+        issueUnixCmdNotSupportedMsg: (profileType: string) =>
+            vscode.l10n.t({
+                message: "Issuing UNIX commands is not supported for this profile type, {0}.",
+                args: [profileType],
+                comment: ["Profile type"],
+            }),
         sshSessionErrorMsg: vscode.l10n.t("Error preparing SSH connection for issuing UNIX commands, please check SSH profile for correctness."),
-        sshProfNotFoundMsg: vscode.l10n.t("No SSH profile found. Please create an SSH profile."),
-        sshProfMissingInfoMsg: vscode.l10n.t("SSH profile missing connection details. Please update."),
-        noProfilesAvailableMsg: vscode.l10n.t("No profiles available."),
         cwdRedirectingMsg: vscode.l10n.t("Redirecting to Home Directory"),
+        sshProfMissingInfoMsg: vscode.l10n.t("SSH profile missing connection details. Please update."),
+        sshProfNotFoundMsg: vscode.l10n.t("No SSH profile found. Please create an SSH profile."),
     };
 
-    public profileInstance = Profiles.getInstance();
-    public outputChannel: vscode.OutputChannel;
+    public sshCwd: string;
     public sshSession: zosuss.SshSession;
     public sshProfile: imperative.IProfileLoaded;
     public isSshRequiredForProf: boolean = false;
 
+    public readonly dialogs: ICommandProviderDialogs = {
+        commandSubmitted: vscode.l10n.t("Unix command submitted."),
+        defaultText: vscode.l10n.t("$(plus) Create a new Unix command"),
+        selectProfile: vscode.l10n.t("Select an SSH profile for this command"),
+        searchCommand: vscode.l10n.t("Enter or update the Unix command"),
+        writeCommand: (options) =>
+            vscode.l10n.t({
+                message: "Select a Unix command to run against {0} (An option to edit will follow)",
+                args: options,
+                comment: ["Current work directory"],
+            }),
+        selectCommand: (options) =>
+            vscode.l10n.t({
+                message: "Select a Unix command to run immediately against {0}",
+                args: options,
+                comment: ["Current work directory"],
+            }),
+    };
+
     public constructor() {
-        super();
-        this.outputChannel = Gui.createOutputChannel(vscode.l10n.t("Zowe Unix Command"));
+        super(vscode.l10n.t("Zowe Unix Command"));
     }
 
     public async issueUnixCommand(node?: IZoweTreeNode, command?: string): Promise<void> {
-        let cwd: string;
+        ZoweLogger.trace("UnixCommandHandler.issueUnixCommand called.");
+
         if (node) {
-            this.serviceProf = node.getProfile();
-            cwd = node.fullPath;
+            // await this.checkCurrentProfile(node);
```

Comment:
Was this commented out during testing? Just verifying whether we still need it.

Reply:
resolved by 54c2f8921dc9e16c9c93446e4437f2c7382b22a4